images/Dockerfile.ubuntu

```diff
@@ -96,58 +98,87 @@ RUN apt-get update \
     tpm2-* \
     && apt-get remove -y unattended-upgrades && apt-get clean && rm -rf /var/lib/apt/lists/*
 
+FROM common AS systemd-boot
+
+FROM common AS grub
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
+    dracut \
+    dracut-network \
+    grub2-common \
+    # this brings grub as a dependency
+    shim-signed \
+    && apt-get remove -y unattended-upgrades && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+FROM systemd-boot AS systemd-boot-amd64
+FROM systemd-boot AS systemd-boot-arm64
+
+FROM grub AS grub-amd64
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
+    # this will probably break 20.04
+    dracut-live \
+    grub2 \
+    grub-efi-amd64-bin \
+    grub-efi-amd64-signed \
+    grub-pc-bin \
+    os-prober \
```

Comment:
We can drop this no? We generate the entries manually and we don't support any other os along Kairos...

Reply:
moved to the non-uki section